ReactiveCocoa/ReactiveCocoa.h

```diff
@@ -73,7 +73,7 @@ FOUNDATION_EXPORT const unsigned char ReactiveCocoaVersionString[];
 	#import <ReactiveCocoa/UITableViewHeaderFooterView+RACSignalSupport.h>
 	#import <ReactiveCocoa/UITextField+RACSignalSupport.h>
 	#import <ReactiveCocoa/UITextView+RACSignalSupport.h>
-
+    #import <ReactiveCocoa/UISearchBar+RACSignalSupport.h>
```

Comment:
hhmm..buuut, in Xcode and AppCode my import indented to match the other imports
![Uploading Screen Shot 2016-09-09 at 23.02.34.png…]()